contrib/sssd.spec.in

```diff
@@ -474,6 +476,19 @@ Provides the D-Bus responder of the SSSD, called the InfoPipe, that allows
 the information from the SSSD to be transmitted over the system bus.
 
 %if 0%{?rhel} == 9
+%if %{use_sssd_user}
+%package polkit-rules
+Summary: Rules for polkit integration for SSSD
+Group: Applications/System
+License: GPL-3.0-or-later
+Requires: polkit >= 0.106
+Requires: sssd-common = %{version}-%{release}
+
+%description polkit-rules
+Provides rules for polkit integration with SSSD. This is required
+for smartcard support.
```

Comment:
Hi, 

would it make sense to add `... if SSSD is running as user 'sssd.'` to better explain when the package is needed?

bye,
Sumit

Reply:
Thank you, added (and rebased).

Reply:
Thanks. Unfortunately it looks there is a second, unrelated commit.

Reply:
> it looks there is a second, unrelated commit

Ouch... it was from #7628, sorry. Fixed.